### PR TITLE
Uri/new note no gambit linux arm

### PR DESCRIPTION
**Pull request by urikirsh**
Before requesting review:
 - Make sure CI is passing
   - Spell check failure may require adding backticks around code or updating `spelling_wordlist.txt`
   - See `README.md` for information about style and markdown syntax
   - If the CI Details link gives a 404, you need to log in to readthedocs.com
 - Add link to generated documentation here
   - you can find this by following the read the docs link from the CI check

Jira ticket: https://certora.atlassian.net/browse/DOC-390
Link to generated documentation: https://certora-certora-prover-documentation--213.com.readthedocs.build/en/213/

### CODE REVIEW

File: docs/gambit/mutation-verifier.md [hunk 1]
```diff
@@ -192,7 +192,7 @@ At the moment, there are a few ways in which `certoraMutate` can fail.
 Here are some suggestions on how to troubleshoot when that happens. 
 We are actively working on mitigating them.
 
-- There are currently no official Gambit binaries for Linux arm machines. That means that Gambit will not be installed with `pip install certora-cli`, and must be {ref}`built from source <build-gambit-from-source>`.
+- There are currently no official Gambit binaries for Linux arm or Windows. That means that Gambit will not be installed with `pip install certora-cli`, and must be {ref}`built from source <build-gambit-from-source>`.
```
Comment (chandrakananandi) [inline docs/gambit/mutation-verifier.md:195]: @urikirsh i don't think this is quite right. We do have binary for linux. @ozcodes led that effort: you can see the code here: https://github.com/Certora/gambit/blob/a97670e0bfe96b6f38bfe002181916a1754283b7/.github/workflows/gambit.yml#L10

**Review by chandrakananandi — CHANGES_REQUESTED** [review]
I left a comment about linux.

EDIT: on never mind, I deleted my comment, you are referring to ARM and I did not read that carefully. My bad.

File: docs/gambit/mutation-verifier.md [hunk 2]
```diff
@@ -192,8 +192,11 @@ At the moment, there are a few ways in which `certoraMutate` can fail.
 Here are some suggestions on how to troubleshoot when that happens. 
 We are actively working on mitigating them.
 
+- There are currently no official Gambit binaries for Linux arm or Windows. That means that Gambit will not be installed with `pip install certora-cli`, and must be {ref}`built from source <build-gambit-from-source>`.
```
Comment (chandrakananandi) [inline docs/gambit/mutation-verifier.md:195]: arm -> ARM?

**Review by chandrakananandi — APPROVED** [review]

File: docs/gambit/mutation-verifier.md (hunk 2 above)
Comment (urikirsh) [reply docs/gambit/mutation-verifier.md]: Great catch, fixed

**Pull request merged**